CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,133 @@
+
+# Contributor Covenant Code of Conduct
```

Comment:
I think this type of the document must be approved by a bigger circle than just author and reviewer. 

It is also too big of task for a reviewer to quickly go through all the links and points of that document.

Reply:
The code of conduct is adapted from https://www.contributor-covenant.org/, and I agree that it should be reviewed by a bigger circle.
The other files I wrote them myself

Reply:
Also, for reviewing, I recommend checking the formatted files on GitHub directly, instead of raw. It makes reading easier